### ISSUE TITLE
Add support for AutoOpen / child database

### ISSUE THREAD

**Issue by fcrozat** [Kunzisoft/KeePassDX#915]
KeePassXC and Keepass supports automatically opening child database using AutoOpen group ( https://keepassxc.org/docs/KeePassXC_UserGuide.html#_automatic_database_opening / https://keepass.info/plugins.html#keeautoexec ).

This feature, if the application supports opening and searching several databases in memory, allows to share credentials between several people easily (without the hassle of KeeShare https://github.com/hicknhack-software/KeeShare which seems a bit less supported in Keepass* ecosystem).

**Comment by gabeweb** [comment]
I agree with the support of this feature. In KeePassXC it's very helpful and it should also be so in KeePassDX (especially because of the limitations of a mobile interface).

**Comment by J-Jamet** [comment]
Linked to #549, depends on the simultaneous opening of databases #209
